benches/coprocessor_executors/selection/util.rs

```diff
@@ -34,12 +40,15 @@ impl Clone for Box<dyn SelectionBencher> {
 /// A bencher that will use normal selection executor to bench the giving expressions.
 pub struct NormalBencher;
 
-impl SelectionBencher for NormalBencher {
+impl<M> SelectionBencher<M> for NormalBencher
```

Comment:
Looks like `criterion::measurement::Measurement` is repeatedly used. Let's import it?

Reply:
Yes! I should have done this.